services/graphite/service.go

```diff
@@ -34,6 +35,7 @@ const (
 	statBatchesTransmitFail = "batchesTxFail"
 	statConnectionsActive   = "connsActive"
 	statConnectionsHandled  = "connsHandled"
+	statPointsNaNFail       = "pointsNaNFail"
```

Comment:
Minor, but important for readability: can you group this with `statPointsParseFail`?